lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -703,20 +703,21 @@ push @tests, __PACKAGE__->new(
 
 # There's no AP uniqueness requirement?
 push @tests, __PACKAGE__->new(
-    test_query => "select *
-                     from ap
-                    where invnumber in (select invnumber
-                                               from ap
-                                              group by invnumber
-                                              having count(*) > 1)
-                    order by invnumber",
-    display_name => $locale->text('Non-unique invoice numbers'),
+    test_query => "SELECT id, concat(invnumber,'-',row_number() over(partition by invnumber order by id)) AS invnumber,
```

Comment:
@ylavoie with this change, we no longer show the user the problematic content. I'd expect that to be confusing more than that this commit is helpful (in the sense that this commit prevents data-entry)